### ISSUE TITLE
User and PW Saving

### ISSUE THREAD

**Issue by seathasky** [Parquelle/WoWRetroLauncher#1]
Should be a feature. No one want's to retype their User and PW every time they log in.

**Comment by Parquelle** [comment]
I don't believe there's a way to launch the game with a log in surpassing the battle.net app. If you tick the checkbox in WoW's log in screen, "Remember this account", you will only need to retype your password upon launch. You can also make a .txt document in the same directory as your Wow.exe file that contains your username and password on the first and second line respectively, that should log you in automatically.

I could implement a feature that does this, but I don't want to be responsible for poorly secured storage of log in information. While typing this comment, I did get an idea though, thank you. :)

**Comment by seathasky** [comment]
I guess that makes sense yea. Security is an issue.
